wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/provisioncredential/EvidenceRequest.kt

```diff
@@ -1242,3 +1244,91 @@ fun AusweisView(
         }
     }
 }
+
+@Composable
+fun EvidenceRequestWebView(
+    evidenceRequest: EvidenceRequestWeb,
+    provisioningViewModel: ProvisioningViewModel,
+    walletServerProvider: WalletServerProvider,
+    documentStore: DocumentStore
+) {
+    val context = LocalContext.current
+    val url = Uri.parse(evidenceRequest.url)
+    val redirectUri = evidenceRequest.redirectUri
+    val scope = rememberCoroutineScope()
+    LaunchedEffect(url, redirectUri) {
+        // NB: these scopes will be cancelled when navigating outside of this screen.
+        val appSupport = walletServerProvider.getApplicationSupport()
+        scope.launch {
+            // Wait for notifications
+            appSupport.notifications.collectLatest { notification ->
+                if (notification.baseUrl == redirectUri) {
+                    handleLanding(appSupport, redirectUri, provisioningViewModel,
+                        walletServerProvider, documentStore)
+                }
+            }
+        }
+
+        // Launch the browser
+        val browserIntent = Intent(Intent.ACTION_VIEW, url)
```

Comment:
Would it make sense to use Chrome Custom Tabs rather than launching the browser?
(see my comment in the design doc)

Reply:
Added a todo. We probably do. Important thing is that we must not use WebView.